timemachine/fe/atom_mapping.py

```diff
@@ -64,9 +115,18 @@ def mcs_map_graph_only_complete_rings(a, b, timeout: int = 3600, smarts: Optiona
         matchChiralTag=False,
         atomCompare=Chem.rdFMCS.AtomCompare.CompareAny,
         bondCompare=Chem.rdFMCS.BondCompare.CompareAny,
+        seedSmarts=smarts if type(smarts) == str else "",
     )
 
 
+def mcs_graph_only_complete_rings(a, b, timeout: int = 3600, smarts: Optional[str] = None):
```

Comment:
We should probably reduce this default timeout. An hour to find MCS seems extreme as a default. 
```suggestion
def mcs_graph_only_complete_rings(a, b, timeout: int = 60, smarts: Optional[str] = None):
```

Reply:
Agreed -- that seems like a good change to introduce.

Is there a reason this timeout should remain higher than for the conformer-aware MCS?

(Before this PR, conformer-aware MCS has timeout = 5 seconds and graph-only MCS has timeout = 1 hr, but perhaps both should = 5 seconds or both = 1 minute.)

Reply:
Applied in https://github.com/proteneer/timemachine/pull/812/commits/354f0dc3efa751a8d2053b0614ebc9f05c1a1977

Reply:
(Side note: increasing timeout from 5 seconds to 1 minute for conformer-aware MCS does not avoid timing out and returning a trivial (1-bond) substructure in the example in https://github.com/proteneer/timemachine/pull/812#issue-1356239660 )

Reply:
Consolidated with default timeout = 5 seconds, and set timeout = 60 seconds in the one usage where conformer_aware=False https://github.com/proteneer/timemachine/pull/812/commits/7588bcc945c7b701540d3bca6f546177853d1650

Reply:
> Is there a reason this timeout should remain higher than for the conformer-aware MCS?

Only suggested greater than 5 because it was larger initially, so there might have been a justification at some point for a longer time out. Like consolidating to 5 seconds and adjusting in example!